### ISSUE TITLE
[BUG] Faulty Image Calibration on Touchscreen

### ISSUE THREAD

**Issue by oxenreport** [oxen-io/session-ios#764]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-ios/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

when zooming in on an image, I double tap on the right and it zooms in on the left side, and when I try to zoom into the top right corner of a photo the behavior is inconsistent with the intended rendering of the image

### Expected Behavior

calibration of images per the side of image selected to zoom in on

### iOS Version

 iPhone 8 iOS version 15.6.1 ( official App Store )

### Session Version

app version 2.2.4 (389)

**Comment by KeeJef** [comment]
Yes, it seems the double tap zoom always zooms in the same location

**Comment by mpretty-cyro** [comment]
Jira ticket SES-1712
